### ISSUE TITLE
Tile geometry building optimization

### ISSUE THREAD

**Issue by gchoqueux** [iTowns/itowns#229]
Tile's geometry are identical if they are in same latitude.
It's possible to reuse the geometry with a rotation matrix.

```js
const rotationY =  (tileCoord.col % 2) * (Math.PI * 2.0 / Math.pow(2, tileCoord.zoom + 1));
tile.rotation.set( 0, rotationY, 0);
```

**Comment by peppsac** [comment]
Actually I think it would be a good idea to fully use three.js transformation matrices when displaying tiles.
The expected result of doing:
```js
tile.add(my_object);
```
would be to have `my_object` on top and at the center (x,y) of `tile`, with it's `Z` axis pointing alongside the normal of `tile`.

**Comment by peppsac** [comment]
(of course if `tile` has an elevation texture we'll need to shift `my_object` up, but this is another issue: #181 )

**Comment by gchoqueux** [comment]
fixed in #586
